### ISSUE TITLE
ImportError: dlopen failed: "/data/data/com.github.andremiras.qrscan/files/app/_python_bundle/site-packages/PI...

### ISSUE THREAD

**Issue by AndreMiras** [AndreMiras/QrScan#27]
https://sentry.io/organizations/andre-5t/issues/1225879526/?referrer=github_plugin

```
ImportError: dlopen failed: "/data/data/com.github.andremiras.qrscan/files/app/_python_bundle/site-packages/PIL/_imaging.so" is 64-bit instead of 32-bit
(1 additional frame(s) were not displayed)
...
  File "/home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/kivy_garden/zbarcam/__init__.py", line 16, in <module>
  File "/home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/kivy_garden/zbarcam/zbarcam.py", line 12, in <module>
  File "/home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/kivy_garden/zbarcam/utils.py", line 2, in <module>
  File "home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/PIL/ImageOps.py", line 20, in <module>
  File "home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/PIL/Image.py", line 64, in <module>

ImportError: dlopen failed: "/data/data/com.github.andremiras.qrscan/files/app/_python_bundle/site-packages/PIL/_imaging.so" is 64-bit instead of 32-bit
(1 additional frame(s) were not displayed)
...
  File "/home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/kivy_garden/zbarcam/__init__.py", line 16, in <module>
  File "/home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/kivy_garden/zbarcam/zbarcam.py", line 12, in <module>
  File "/home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/kivy_garden/zbarcam/utils.py", line 2, in <module>
  File "home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/PIL/ImageOps.py", line 20, in <module>
  File "home/andre/workspace/QrScan/.buildozer/android/platform/build/build/python-installs/qrscan/PIL/Image.py", line 64, in <module>
```

**Comment by AndreMiras** [comment]
Sentry issue: [QRSCAN-Y](https://sentry.io/organizations/andre-5t/issues/1225879532/?referrer=github_plugin)

**Comment by AndreMiras** [comment]
Fixed in last release d169e79bcbee7b6049e93ee556a24bce11857d98 also available on the Play store
